CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Fix the issue when validation on empty union will fail
```

Comment:
I would recommend to rephrase this description.

Reply:
I changed it in different way.